tests/page-objects/default/common/common.wdio.page.js

```diff
@@ -174,7 +174,7 @@ const waitForLoaders = async () => {
 const waitForPageLoaded = async () => {
   // if we immediately check for app loaders, we might bypass the initial page load (the bootstrap loader)
   // so waiting for the main page to load.
-  await (await $('.header-logo')).waitForDisplayed();
+  await (await $('#header-dropdown-link')).waitForDisplayed();
```

Comment:
I think the desired outcome is the same and it's present in both desktop and mobile.